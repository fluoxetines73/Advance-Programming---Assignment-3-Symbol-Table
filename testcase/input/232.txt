BEGIN
BEGIN
END
INSERT x string
END
PRINT
INSERT d string
BEGIN
INSERT x number
PRINT
PRINT
PRINT
END
PRINT