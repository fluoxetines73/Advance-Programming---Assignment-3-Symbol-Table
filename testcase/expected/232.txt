success

success
success
d//0 x//1
d//0 x//1
d//0 x//1
d//0